crowbar_framework/lib/crowbar/installer.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  class Installer
+    class << self
+      def install
+        crowbar_dir = Rails.root.join("..")
+        case NodeObject.admin_node[:platform_family]
+        when "suse"
+          cmd = "sudo #{crowbar_dir}/bin/install-chef-suse.sh --crowbar"
+        else
+          return {
+            status: 501,
+            msg: I18n.t(".system_not_supported", scope: "installer.index")
+          }
+        end
+
+        # spawn the script asynchronously in the background
+        pid = spawn(cmd)
+        Process.detach(pid)
+
+        {
+           status: 200,
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.